### ISSUE TITLE
Missing links in ephemera-METS-file to year-METS-files 

### ISSUE THREAD

**Issue by andre-hohmann** [kitodo/kitodo-production#3633]
### Problem
In the exported METS-file "ephemera", the links to the year-elements are missing in the following METS-sections: 
- <mets:dmdSec> 
- <mets:structMap TYPE="LOGICAL"> 

Probably, this happens for processes of newspapers, too. 

### Solution 
The links must be implemented in the before mentioned file sections. 

#### Examples 3.x
- DresKr_880547324.xml

<details>
  <summary>Click to show the exported METS-File:</summary>

```xml
<mets:mets xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:kitodo="http://meta.kitodo.org/v1/" xmlns:mets="http://www.loc.gov/METS/" xmlns:dv="https://dfg-viewer.de/profil-der-metadaten/" xmlns:mods="http://www.loc.gov/mods/v3" xmlns:slub="http://www.slub-dresden.de/namespace">
	<mets:metsHdr CREATEDATE="2020-05-17T08:56:16.119+02:00" LASTMODDATE="2020-05-17T10:36:07.131+02:00">
		<mets:metsDocumentID>1461</mets:metsDocumentID>
	</mets:metsHdr>
	<mets:dmdSec ID="uuid-9882ed85-222d-3ed8-b0cb-bc6ee5c2741e">
		<mets:mdWrap>
			<mets:xmlData>
				<mods:mods>
					<mods:titleInfo>
						<mods:title>Dresdner Kreuzchor</mods:title>
						<mods:subTitle>[Programmhefte]</mods:subTitle>
					</mods:titleInfo>
					<mods:originInfo eventType="publication">
						<mods:place>
							<mods:placeTerm type="text">Dresden</mods:placeTerm>
						</mods:place>
						<mods:dateIssued keyDate="yes">[1889-1945]</mods:dateIssued>
						<mods:dateIssued type="order" encoding="w3cdtf">1889</mods:dateIssued>
					</mods:originInfo>
					<mods:language>
						<mods:languageTerm type="code" authority="rfc3066">ger</mods:languageTerm>
					</mods:language>
					<mods:physicalDescription>
						<mods:extent>Online-Ressource</mods:extent>
						<mods:digitalOrigin>reformatted digital</mods:digitalOrigin>
					</mods:physicalDescription>
					<mods:identifier type="zdb">2885004-X</mods:identifier>
					<mods:recordInfo>
						<mods:recordIdentifier source="http://digital.slub-dresden.de/oai/">oai:de:slub-dresden:db:id-880547324</mods:recordIdentifier>
					</mods:recordInfo>
					<mods:identifier type="purl">http://digital.slub-dresden.de/id880547324</mods:identifier>
					<mods:identifier type="urn">urn:nbn:de:bsz:14-db-id880547324</mods:identifier>
					<mods:identifier type="swb-ppn-digital">880547324</mods:identifier>
					<mods:identifier type="swb-ppn">880547324</mods:identifier>
					<mods:relatedItem type="series">
						<mods:titleInfo>
							<mods:title lang="ger">Musik</mods:title>
						</mods:titleInfo>
					</mods:relatedItem>
					<mods:relatedItem type="series">
						<mods:titleInfo>
							<mods:title lang="ger">LDP: Performance Ephemera</mods:title>
						</mods:titleInfo>
					</mods:relatedItem>
					<mods:relatedItem type="series">
						<mods:titleInfo>
							<mods:title lang="ger">Saxonica</mods:title>
						</mods:titleInfo>
					</mods:relatedItem>
					<mods:relatedItem type="series">
						<mods:titleInfo>
							<mods:title lang="ger">Performance Ephemera</mods:title>
						</mods:titleInfo>
					</mods:relatedItem>
					<mods:extension>
						<slub:slub>
							<slub:id type="digital">880547324</slub:id>
							<slub:id type="tsl-ats">DresKr</slub:id>
						</slub:slub>
					</mods:extension>
				</mods:mods>
			</mets:xmlData>
		</mets:mdWrap>
	</mets:dmdSec>
	<mets:amdSec>
		<mets:techMD ID="uuid-1d0ef207-3a8d-342a-bfb7-a7efabfc59e8">
			<mets:mdWrap>
				<mets:xmlData>
					<mods:mods/>
				</mets:xmlData>
			</mets:mdWrap>
		</mets:techMD>
		<mets:rightsMD ID="uuid-33f93773-023f-3acb-bd87-219489b1a342">
			<mets:mdWrap>
				<mets:xmlData>
					<dv:rights>
						<dv:owner>Sächsische Landesbibliothek - Staats- und Universitätsbibliothek Dresden</dv:owner>
						<dv:ownerLogo>http://digital.slub-dresden.de/fileadmin/images/dfgviewer_logo_slub.gif</dv:ownerLogo>
						<dv:ownerSiteURL>http://www.slub-dresden.de/</dv:ownerSiteURL>
						<dv:ownerContact>mailto:digital@slub-dresden.de</dv:ownerContact>
					</dv:rights>
				</mets:xmlData>
			</mets:mdWrap>
		</mets:rightsMD>
		<mets:digiprovMD ID="uuid-04e81c4a-8970-37c8-8527-23619e8737fb">
			<mets:mdWrap>
				<mets:xmlData>
					<dv:links>
						<dv:presentation>https://digital.slub-dresden.de/id880547324</dv:presentation>
						<dv:reference>http://dienste.slub-dresden.de/cgi-bin/FOZK.pl?PPN=880547324</dv:reference>
					</dv:links>
				</mets:xmlData>
			</mets:mdWrap>
		</mets:digiprovMD>
	</mets:amdSec>
	<mets:fileSec/>
	<mets:structMap TYPE="PHYSICAL">
		<mets:div ID="uuid-1e6d7284-222d-417c-aaf0-1c98d2ecc35c"/>
	</mets:structMap>
	<mets:structMap TYPE="LOGICAL">
		<mets:div ID="uuid-d9b2d075-302c-42d9-b797-2fece738b2d5" DMDID="uuid-9882ed85-222d-3ed8-b0cb-bc6ee5c2741e" ADMID="uuid-04e81c4a-8970-37c8-8527-23619e8737fb uuid-33f93773-023f-3acb-bd87-219489b1a342 uuid-1d0ef207-3a8d-342a-bfb7-a7efabfc59e8" TYPE="ephemera" ORDER="1" ORDERLABEL="Dresdner Kreuzchor" LABEL="Dresdner Kreuzchor"/>
	</mets:structMap>
	<mets:structLink/>
</mets:mets>
```
</details>

#### Examples 2.x
- https://digital.slub-dresden.de/data/kitodo/DresKr_880547324-1936091902_02-v/DresKr_880547324-1936091902_02-v_anchor.xml

**Comment by matthias-ronge** [comment]
Looks like this is another duplicate of #3623

**Comment by andre-hohmann** [comment]
Closed, because duplicate to #3623.
